app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -103,6 +103,41 @@ def test_create_bugzilla_ticket_retries_with_no_assignee(self):
             settings.BUGZILLA_CREATE_URL, expected_call_data
         )
 
+    def test_create_bugzilla_ticket_retries_with_no_cf_tracking(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT, name="An Experiment"
+        )
+        self.setUpMockBugzillaInvalidFirefoxVersion()
+
+        response_data = create_experiment_bug(experiment)
```

Comment:
I would call this variable `bugzilla_id` so it's clear it's the ID and not the entire response body.